Doc/whatsnew/3.7.rst

```diff
@@ -652,103 +1288,189 @@ can be set within the scope of a group.
 (Contributed by Christian Heimes in :issue:`32951`)
 
 OpenSSL 1.1 APIs for setting the minimum and maximum TLS protocol version are
-available as :attr:`~ssl.SSLContext.minimum_version` and
-:attr:`~ssl.SSLContext.maximum_version`. Supported protocols are indicated
-by new flags like :data:`~ssl.HAS_TLSv1_1`.
+available as :attr:`SSLContext.minimum_version <ssl.SSLContext.minimum_version>`
+and :attr:`SSLContext.maximum_version <ssl.SSLContext.maximum_version>`.
+Supported protocols are indicated by serveral new flags, such as
+:data:`~ssl.HAS_TLSv1_1`.
 (Contributed by Christian Heimes in :issue:`32609`.)
 
+
 string
 ------
 
 :class:`string.Template` now lets you to optionally modify the regular
 expression pattern for braced placeholders and non-braced placeholders
 separately.  (Contributed by Barry Warsaw in :issue:`1198569`.)
 
+
 subprocess
 ----------
 
+The :func:`subprocess.run` function accepts the new *capture_output*
+keyword argument.  When ``True``, stdout and stderr will be captured.
+This is equivalent to passing :data:`subprocess.PIPE` as *stdout* and
+*stderr* arguments.
+(Contributed by Bo Bayles in :issue:`32102`.)
+
+The ``subprocess.run`` function and the :class:`subprocess.Popen` constructor
+now accept the *text* keyword argument as an alias
+to *universal_newlines*.
+(Contributed by Andrew Clegg in :issue:`31756`.)
+
 On Windows the default for *close_fds* was changed from ``False`` to
 ``True`` when redirecting the standard handles.  It's now possible to set
 *close_fds* to ``True`` when redirecting the standard handles.  See
-:class:`subprocess.Popen`.
+:class:`subprocess.Popen`.  This means that *close_fds* now defaults to
+``True`` on all supported platforms.
+(Contributed by Segev Finer in :issue:`19764`.)
+
+The subprocess module is now more graceful when handling
+:exc:`KeyboardInterrupt` during :func:`subprocess.call`,
+:func:`subprocess.run`, or in a :class:`~subprocess.Popen`
+context manager.  It now waits a short amount of time for the child
+to exit, before continuing the handling of the ``KeyboardInterrupt``
+exception.
+(Contributed by Gregory P. Smith in :issue:`25942`.)
 
-This means that *close_fds* now defaults to ``True`` on all supported
-platforms.  (Contributed by Segev Finer in :issue:`19764`.)
 
 sys
 ---
 
 Added :attr:`sys.flags.dev_mode` flag for the new development mode.
 
-tkinter
--------
+The new :func:`sys.breakpointhook` hook function is called by the
+built-in :func:`breakpoint`.
+(Contributed by Barry Warsaw in :issue:`31353`.)
+
+On Android, the new :func:`sys.getandroidapilevel` returns the build-time
+Android API version.
+(Contributed by Victor Stinner in :issue:`28740`.)
+
+The new :func:`sys.get_coroutine_origin_tracking_depth` function returns
+the current coroutine origin tracking depth, as set by
+the new :func:`sys.set_coroutine_origin_tracking_depth`.
+(Contributed by Nathaniel J. Smith in :issue:`32591`.)
```

Comment:
I'd also mention here that asyncio now uses this new mechanism, and that this new API is intended to replace `sys.set_coroutine_wrapper()` API, which is, btw, deprecated in 3.7.